apcd-cms/src/apps/registrations/templates/submission_form/registration_form_body.html

```diff
@@ -581,16 +467,16 @@ <h4>
             <div class="field-wrapper textinput required">
                 <div class="field-errors" style="display: none"></div>
 
-                <label for="entity_name">
+                <label for="entity_name_1">
                 Name<span class="asterisk">*</span>
                 </label>
 
                 <input
                 type="text"
-                name="entity_name"
+                name="entity_name_1"
                 required
                 class="textinput"
-                id="entity_name"
+                id="entity_name_1"
```

Comment:
One example of labels that have changed between staging and prod deployments.

Reply:
I edited all the queries used by the registration app's view so that they match the staging queries. Here are the modified queries. 

- create_registration
- create_registration_entity
- create_registration_contact